src/com/company/exercises/Demo.java

```diff
@@ -0,0 +1,36 @@
+package com.company.exercises;
+
+public class Demo {
+    @FunctionalInterface
+    public interface StringRunnable {
+        public abstract void run(String str);
```

Comment:
Methods in interfaces are already public abstract by default.

Reply:
Done.